src/Video.tsx

```diff
@@ -524,14 +525,24 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
         if (data?.spcBase64) {
           try {
             // Handles both scenarios, getLicenseOverride being a promise and not.
-            const license = await drm.getLicense(
+            const license = await selectedDrm.getLicense(
               data.spcBase64,
               data.contentId,
               data.licenseUrl,
               data.loadedLicenseUrl,
             );
-            result =
-              typeof license === 'string' ? license : 'Empty license result';
+            if (typeof license === 'string') {
+              if (nativeRef.current) {
+                NativeVideoManager.setLicenseResultCmd(
+                  getReactTag(nativeRef),
+                  license,
+                  data.loadedLicenseUrl,
+                );
+              }
+              return;
+            } else {
+              result = 'Empty license result';
+            }
```

Comment:
```ts
            if (typeof license === 'string') {
              if (nativeRef.current) {
                NativeVideoManager.setLicenseResultCmd(
                  getReactTag(nativeRef),
                  license,
                  data.loadedLicenseUrl,
                );
              }
              return;
            } else {
              result = 'Empty license result';
            }
```
I'm not sure above change is really necessary.

Reply:
Without it, setLicenseResultCmd() was never called - the licence is never imported, and therefore playback never starts

Reply:
https://github.com/TheWidlarzGroup/react-native-video/pull/4042/files#diff-272334bc4a560000e8736860b4e6980d3f865b9abea3baee3bfee41ab1894bd6R552-R558
Not called from above?

Reply:
That's calling setLicenseResult**Error**Cmd() - i.e. indicating that the licence call failed.

Reply:
I get it! thank you👍

Reply:
nit;
I've rewrite some code to make it a bit improve readability. It's just suggestion. you don't have to apply it :) Thank you for contributing.
```ts
        try {
          if (!data?.spcBase64) throw new Error('No spc received');
          // Handles both scenarios, getLicenseOverride being a promise and not.
          const license = await Promise.resolve(selectedDrm.getLicense(
            data.spcBase64,
            data.contentId,
            data.licenseUrl,
            data.loadedLicenseUrl,
          )).catch(() => { throw new Error('fetch error') });
          if (typeof license !== 'string') throw Error('Empty license result')
          if (nativeRef.current) {
            NativeVideoManager.setLicenseResultCmd(
              getReactTag(nativeRef),
              license,
              data.loadedLicenseUrl,
            );
          }
        } catch (e) { 
          const msg = e instanceof Error ? e.message : 'fetch error'
          if (nativeRef.current) {
            NativeVideoManager.setLicenseResultErrorCmd(
              getReactTag(nativeRef),
              msg,
              data.loadedLicenseUrl,
            );
          }
        }
```